src/main/scala/com/fulcrumgenomics/basecalling/ExtractIlluminaRunInfo.scala

```diff
@@ -27,74 +27,67 @@ package com.fulcrumgenomics.basecalling
+    |1. run_barcode: the unique identifier for the sequencing run and flowcell, stored as "<instrument-name>_<flowcell-barcode>".
+    |2. flowcell_barcode: the flowcell barcode.
+    |3. instrument_name: the instrument name.
+    |4. run_date: the date of the sequencing run.
+    |5. read_structure: the description of the logical structure of cycles within the sequencing run, including which cycles
     |   correspond to sample barcodes, molecular barcodes, template bases, and bases that should be skipped.
-    |5. Number of Lanes: the number of lanes in the flowcell.
+    |6. number_of_lanes: the number of lanes in the flowcell.
   """)
 class ExtractIlluminaRunInfo
 (
   @arg(flag="i", doc="The input RunInfo.xml typically found in the run folder.") val input: FilePath,
-  @arg(flag="o", doc="The output file.") val output: FilePath,
-  @arg(flag="d", doc="The delimiter to use") val delim: String = "\t"
+  @arg(flag="o", doc="The output file.") val output: FilePath
 ) extends FgBioTool {
 
   Io.assertReadable(input)
   Io.assertCanWriteFile(output)
 
   override def execute(): Unit = {
-    val info = RunInfo(runInfo=input)
-    val lines = Seq(
-      ExtractIlluminaRunInfo.HeaderColumns,
-      Seq(info.runBarcode, info.flowcellBarcode, info.runDate, info.readStructure, info.numLanes).map(_.toString)
-    ).map(_.mkString(delim))
-    Io.writeLines(output, lines)
+    Metric.write(output, RunInfo(runInfo=input))
   }
 }
 
-object ExtractIlluminaRunInfo {
-  val HeaderColumns: Seq[String] = Seq("run_barcode", "flowcell_barcode", "run_date", "read_structure", "num_lanes")
-}
-
 /** Stores the result of parsing the run info file.
   *
-  * @param runBarcode the unique identifier for the sequencing run and flowcell, stored as
+  * @param run_barcode the unique identifier for the sequencing run and flowcell, stored as
   *                   "<instrument-name>_<flowcell-barcode>".
-  * @param flowcellBarcode the flowcell barcode.
-  * @param runDate the date of the sequencing run.
-  * @param readStructure the description of the logical structure of cycles within the sequencing run, including which cycles
+  * @param flowcell_barcode the flowcell barcode.
+  * @param instrument_name the instrument name.
+  * @param run_date the date of the sequencing run.
+  * @param read_structure the description of the logical structure of cycles within the sequencing run, including which cycles
   *                      correspond to sample barcodes, molecular barcodes, template bases, and bases that should be skipped.
-  * @param numLanes the number of lanes in the flowcell.
+  * @param num_lanes the number of lanes in the flowcell.
   */
 case class RunInfo
-( runBarcode: String,
-  flowcellBarcode: String,
-  runDate: Iso8601Date,
-  readStructure: ReadStructure,
-  numLanes: Int
-)
+( run_barcode: String,
+  flowcell_barcode: String,
+  instrument_name: String,
+  run_date: Iso8601Date,
+  read_structure: String, // NB: would like this to be ReadStructure
```

Comment:
Does it not work?  Will Metric not toString it and use the `apply(String)` to re-hydrate it?